staging/src/k8s.io/apiserver/pkg/server/patch_genericapiserver.go

```diff
@@ -80,6 +81,7 @@ func WithLateConnectionFilter(handler http.Handler) http.Handler {
 
 		if late {
 			if pth := "/" + strings.TrimLeft(r.URL.Path, "/"); pth != "/readyz" && pth != "/healthz" && pth != "/livez" {
+				metrics.RequestLateGracefulTerminationTotal.Inc()
```

Comment:
The loopback client (line 85) is a bit special and doesn't follow `/readyz` protocol. I think we could exclude this particular client from the metrics. That way the metric would tract only external clients. The same applies to `RequestBeforeReadyTotal`